CHANGELOG.rst

```diff
@@ -11,13 +11,18 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
+[2.1.7] - 2020-01-28
+--------------------
+
+* Revert Initialized EnrollmentApiClient with enterprise service worker user
```

Comment:
Can we change the statement to "Revert EdX-API-KEY replacement changes in Enrollment, Course, and ThirdPartyAuth client"